### PR TITLE
Control Flow Statements

### PR DESCRIPTION
**Pull request by AmrutaDhage20**

### CODE REVIEW

**Review by prashantkawa — APPROVED** [review]
Approving PR

File: src/com/encora/javaAssignments/ControlFlowStatements/IfElseIfStatement.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Program for IfElseIf Statement:
+package ControlFlowStatements;
+
```
Comment (amitsahu8) [inline src/com/encora/javaAssignments/ControlFlowStatements/IfElseIfStatement.java:2]: please follow the coding guideline points, go through it properly

File: src/com/encora/javaAssignments/ControlFlowStatements/IfElseStatement.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+//Program for IfElse statement:
+package ControlFlowStatements;
```
Comment (amitsahu8) [inline src/com/encora/javaAssignments/ControlFlowStatements/IfElseStatement.java:1]: above same comment for all the files

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: src/com/encora/javaAssignments/ControlFlowStatements/IfStatement.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    //Program for If Statement:
+
+    public static void student(int rank) {
+
+        if (rank < 0) {
+            System.out.println("The number is Negative");
+        }
+        System.out.println("The number is Positive");
+    }
```
Comment (prashantkawa) [inline src/com/encora/javaAssignments/ControlFlowStatements/IfStatement.java]: For negative input both the statements will get printed, please change the logic

File: src/com/encora/javaAssignments/ControlFlowStatements/NestedIfDemo.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
+package ControlFlowStatements;
+
+public class NestedIfDemo {
+
+    // Program using NestedIf :
+
+    public static void compareNumbers(int integerNumber) {
+        // int integerNumber = 50;
```
Comment (prashantkawa) [inline src/com/encora/javaAssignments/ControlFlowStatements/NestedIfDemo.java]: remove commented code

**Review by prashantkawa — CHANGES_REQUESTED** [review]

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**